README.md

```diff
@@ -102,34 +48,49 @@ Run a specific `gohack` version:
 {{PrintBlock "gohack run" -}}
 ```
 
-
 -->
 
+### Installation
+
+```
+$ GO111MODULE=off go get -u github.com/myitcv/gobin
```

Comment:
Yes, I think we do. Because otherwise, if the user is in the context of a module, it will modify that `go.mod` which is the whole complaint raised in https://github.com/golang/go/issues/27643. I definitely want to side-step that here.